storybookjs/storybook
# Make app/react/package.json private to not leak dependencies

Issue: Previously the dependencies of the react app were leaking into my project

## What I did

set the app/react/package.json to private


## How to test

in an app that uses storybook run `yarn install` and no longer see the warning below

`@kadira/storybook > babel-preset-react-app > babel-preset-latest@6.16.0: 💥  preset-latest accomplishes the same task as babel-preset-env. 🙏  Please install it with 'npm install babel-preset-env --save-dev'. '{ "presets": ["latest"] }' to '{ "presets": ["env"] }'. For more info, please check the docs: http://babeljs.io/docs/plugins/preset-env 👌. And let us know how you're liking Babel at @babeljs on 🐦`
